contrib/experiments/interpretation/penobscot/local/test.py

```diff
@@ -268,9 +268,9 @@ def _tensor_to_numpy(pred_tensor):
 
     logger.info("Starting training")
     if debug:
-        evaluator.run(test_loader, max_epochs=1, epoch_length = 1)
+        evaluator.run(test_loader, max_epochs=1, epoch_length=1)
```

Comment:
Shouldn't this be config.TRAIN.BATCH_SIZE_PER_GPU for epoch_length?

Reply:
This seems to be the same as the latest code from the correctness branch (after the ignite upgrade). Here's the code:

https://github.com/microsoft/seismic-deeplearning/blob/cf336ee059612e621550475e34bc943b0498dbaa/contrib/experiments/interpretation/penobscot/local/test.py#L286-L288

Reply:
OK then let's just leave it as is